src/platform/common/types.ts

```diff
@@ -350,3 +350,33 @@ export interface ICellRange {
     range: Range;
     cell_type: string;
 }
+
+// Used by platform/common/scriptPathUtils
+export const IScriptPathUtils = Symbol('IConfigurationService');
+export interface IScriptPathUtils {
+    rootDirectory: string;
```

Comment:
i world remove this, and use `context.extensionUri` where ever this is required, else we just end up having more than one way to get the same value

Reply:
i dont think this is used anywhere, hence you can remove this.

Reply:
Just to confirm, `context.extensionPath` is valid in this case. Is that correct?

Reply:
Yes, however if this is going to be used on web as well, then you would want to use `context.extensionUri`

Reply:
Even then, I need to do `uri.path` at some point to get the string version of the URI 🤔

Reply:
Ohhh `extensionPath` is shorthand of `extensionUri.fsPath`.... I expected it to be shorthand of `extensionUri.path` 🤦‍♂️ fixing it!